### PR TITLE
PRI-106 - fix: sql injection by parsing to number

### PR DESCRIPTION
**Pull request by kgmyatthu**

### CODE REVIEW

File: src/routes/log.ts [hunk 1]
```diff
@@ -121,8 +121,8 @@ const prepareSQLFilters = ({
 }
 router.route('/api-stats').get(async (req: CustomRequest, res: Response) => {
   try {
-    const page = req.query.page || 0
-    const max = req.query.max || 5000
+    const page = Number(req.query.page) || 0
```
Comment (mhanson-github) [inline src/routes/log.ts:124]: Is the valid range of page & max?  Can they be passed in with negative values?  non-integer values?

**Review by arhamj — CHANGES_REQUESTED** [review]
@kgmyatthu can we also make the query parameterised using either a named param or a ? unnamed param?

**Comment by kgmyatthu** [comment]
> @kgmyatthu can we also make the query parameterised using either a named param or a ? unnamed param?

I remember trying this with the sql package that's currently in use. It didn't work. I'll revisit them again.
UPDATE: Ok we had a internal discussion. Parameterized queries will come later with separate sweep operation. Right now we agreed to just put targeted fixes at the input level.

File: src/routes/log.ts (hunk 1 above)
Comment (kgmyatthu) [reply src/routes/log.ts:124]: Good point I'll revisit, My guess is math will break and client doesn't get anything.

**Review by arhamj — DISMISSED** [review]

**Review by afostr — DISMISSED** [review]
looks good thanks

**Review by mhanson-github — APPROVED** [review]

**Pull request merged**